deepfence_worker/cronjobs/neo4j.go

```diff
@@ -130,10 +130,10 @@ func CleanUpDB(ctx context.Context, task *asynq.Task) error {
 
 	// registry images
 	if _, err = session.Run(`
-		MATCH (n:ContainerImage)
-		WHERE exists((n)<-[:HOSTS]-(:RegistryAccount))
-		AND n.updated_at < TIMESTAMP()-$time_ms
+		MATCH (n:ContainerImage)<-[:HOSTS]-(m:RegistryAccount)
+		WHERE n.updated_at < TIMESTAMP()-$time_ms
 		AND n.active = true
+		AND m.last_synced_at >= m.updated_at
```

Comment:
We should use `m.updated_at` instead of `m.last_synced_at`
A `:RegistryAccount` node should be updated after each sync only?

Reply:
not necessarily, `updated_at` in `:RegistryAccount`  update even when we update `syncing` to true or false

Reply:
Ok, let's be explicit and change:
```
AND m.last_synced_at >= m.updated_at
```
into:
```
AND m.last_synced_at = m.updated_at
```
The > scenario being impossible